bodhi/server/models.py

```diff
@@ -790,8 +793,7 @@ def version_int(self):
         Returns:
             int: The version of the release.
         """
-        regex = re.compile(r'\D+(\d+)[CMF]?$')
-        return int(regex.match(self.name).groups()[0])
+        return int(self._version_int_regex.match(self.name).groups()[0])
```

Comment:
I am curious, why this change was needed ?

Reply:
It's a (maybe a little over the top) optimization: previously, we manually compiled the regex every time the `version_int` property is accessed. Because the regex is constant, we can just do it once and reuse it.